src/image_processing/process_node.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
What do we plan to do with this node @shubhankar12? Will it handle all of our image processing? Won't we need a specific node for each task?

Reply:
@abrandemuehl  I still have to think about this. We will be using YOLO and that would be common for everything. But I will get back to you on this.